Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+             result.
+     """
+    sha2 = args.get('file', '')
+
+    response_verdict = client.get_sandbox_verdict_for_file(sha2)
+    file_res = client.get_file_entity(sha2)
+
+    response_verdict |= file_res
+    # Sandbox verdict
+    title = "Sandbox Verdict"
+    if response_verdict:
+        readable_output = generic_readable_output([response_verdict], title)
+    else:
+        readable_output = f'{title} does not have data to present. \n'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.SandboxVerdict',
+        outputs_key_field='',
+        outputs=response_verdict,
+        raw_response=response_verdict
+    )
+
+
+def check_sandbox_status(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+     Query file Sandbox command status,
+     Args:
+         client: client object to use.
+         args: all command arguments, usually passed from ``demisto.args()``.
+     Returns:
+         CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+             result.
+    """
+    readable_data = []
+    title = "File Sandbox Status"
+    if not (command_id := args.get('command_id')):
+        raise DemistoException('Command ID missing.')
+
+    response = client.get_sandbox_status(command_id)
+    # Query Sandbox Command Status
+    summary_data = {}
+    if sandbox_status := response.get("status"):
+        for status in sandbox_status:
+            summary_data = {
+                'command_id': command_id,
+                'status': SANDBOX_STATE.get(str(status.get('state', ''))),
+                'message': status.get('message', ''),
+                'target': status.get('target', ''),
+                'error_code': status.get('error_code', '')
+            }
+
+    if summary_data:
+        readable_data.append(summary_data)
+        readable_output = generic_readable_output(readable_data, title)
+    else:
+        readable_output = f'{title} does not have data to present. \n'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.SandboxStatus',
+        outputs_key_field='command_id',
+        outputs=summary_data,
+        raw_response=response
+    )
+
+
+def issue_sandbox_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+     Issue File Sandbox command,
+     Args:
+         client: client object to use.
+         args: all command arguments, usually passed from ``demisto.args()``.
+     Returns:
+         CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+             result.
+     """
+
+    sha2 = args.get('file', '')
+    if not re.match(sha256Regex, sha2):
+        raise ValueError(f'SHA256 value {sha2} is invalid')
```

Comment:
we have a method that does this, let's use it instead of duplicating the code.

Reply:
Using this method "get_hash_type(sha2)" from CommonServerPython